tests/test_base_models.py

```diff
@@ -485,6 +518,38 @@ def test_fit_predict_multi_model(self):
         post_load_preds = model.predict(test_texts)
         self.assertEqual(post_load_preds, pre_save_preds)
 
+    def test_featurize_sequence(self):
```

Comment:
It looks like the content of these tests is the same except for the source_model used? Could we pull out the shared code here?